nss-day-cohort-48/giffygram-altavista-giffygram
# Jf dm form

1. Created HTML elements for the Direct Message Form.
2. Created clickEvents to capture the message and used the messageProvider to send to API.
3. Rendered it by interpolated on GiffyGram module, may need to be moved later in the process.    ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout jf-DM-form
   ```
2. Open a new Terminal and open the giffygram repo (make sure it's named 'giffygram')
3. Run `./serve-giffygram` to serve the code.
4. Test app functionality.
   > check and see that the form renders on the DOM
   >make sure that the message is being stored in the API 
  > make sure the cancel button is working as well. It should state a change.

5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
